counter-app/src/main/java/camp/nextstep/edu/counter/Counter.kt

```diff
@@ -0,0 +1,19 @@
+package camp.nextstep.edu.counter
+
+class Counter {
+    var number = 0
+        private set
+
+    fun increment() {
+        number++
+    }
+
+    fun decrement() {
+        if (number > 0) {
+            number--
+            return
+        }
+
+        throw IllegalStateException("0 이하로 내릴 수 없습니다")
```

Comment:
현재는 정상적인 decrement 함수의 기능을 분기처리하여 얼리 리턴하는 구조네요,

decrement 함수의 메인 기능을 메인 블록에 작성 하고,
예외 처리를 분기문으로 얼리리턴하는 구조는 어떨까요?
```suggestion
        if (number <= 0) {
               throw IllegalStateException("0 이하로 내릴 수 없습니다")
        }
        number--

```

Reply:
추가로 해당 분기처리는 코틀린에서 제공하는 check, require 을 활용해보면 어떨까요?

Reply:
분기로 검증하는 로직이 먼저 있는게 일반적이여 보이네욥!
kotlin의 문법을 활용해서 수정해보겠습니다!

감사합니다 :)